examples/java-matter-controller/java/src/com/matter/controller/commands/common/MutableInteger.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+package com.matter.controller.commands.common;
+
+public class MutableInteger {
```

Comment:
Perhaps `AtomicInteger` can be used here?

Reply:
AtomicInteger wrap an int value in it, and does not take boolean value. We need a wrapper class which could store a value with type of short, int, long, and boolean.

Reply:
AtomicInteger can hold both an int or short. AtomicLong also exists, as well as AtomicBoolean